perception/map_based_prediction/CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ if(NOT CMAKE_CXX_STANDARD)
   set(CMAKE_CXX_EXTENSIONS OFF)
 endif()
 if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
-  add_compile_options(-Wall -Wextra -Wpedantic -Werror)
```

Comment:
Is it ok to remove `-Werror`?

Reply:
There are some unused arguments in the code, which are ignored because `-Werror` was removed.

Reply:
done

Reply:
@purewater0901 cc @takayuki5168 
It has been erased, is that ok?
What does done mean?

Reply:
@yukkysaito Yes, I've removed this part.

Reply:
@purewater0901 Why did you remove it?

Reply:
I thought we don't need this option. Should I add these options?

Reply:
> @yukkysaito I thought we don't need this option. Should I add these options?

@purewater0901 
yes. we need this

Reply:
Actually, we do need `-Werror` to guarantee minimal code quality and avoid trivial errors.

Reply:
@yukkysaito @kenji-miyake Okay, I'll add these options.

Reply:
I finished adding this option.